utils/vscode/language-configuration.json

```diff
@@ -3,22 +3,30 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 {
-  "comments": {
-    "lineComment": "//"
-  },
-  "brackets": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ],
-  "autoClosingPairs": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ],
-  "surroundingPairs": [
-    ["(", ")"],
-    ["[", "]"],
-    ["{", "}"]
-  ]
+    "comments": {
```

Comment:
Our general style is to use 2-space indent, rather than 4. Please can you update this file to follow that?